### PR TITLE
feat(app, shared-data): Add liquidProbe command text

### PR DESCRIPTION
**Pull request by mjhuff**
Closes [EXEC-567](https://opentrons.atlassian.net/browse/EXEC-567)

# Overview

Add new command text for the `liquidProbe` command. I've looked at some similar commands for inspiration. 

# Test Plan
- Covered by new tests.

# Changelog
- Added `liquidProbe` command text.

# Risk assessment
low

[EXEC-567]: https://opentrons.atlassian.net/browse/EXEC-567?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]

File: app/src/molecules/Command/hooks/useCommandTextString/index.tsx [hunk 1]
```diff
@@ -78,6 +78,14 @@ export function useCommandTextString(
         commandText: utils.getLoadCommandText(fullParams),
       }
 
+    case 'liquidProbe':
```
Comment (SyntaxColoring) [inline app/src/molecules/Command/hooks/useCommandTextString/index.tsx:81]: We'll need a case for `tryLiquidProbe`, too. I think for the purposes of the run log, the logic and text can be identical.

**Review by SyntaxColoring — APPROVED** [review]

File: app/src/molecules/Command/hooks/useCommandTextString/index.tsx (hunk 1 above)
Comment (mjhuff) [reply app/src/molecules/Command/hooks/useCommandTextString/index.tsx:81]: @SyntaxColoring, it looks like `tryLiquidProbe` wasn't added to `shared-data`. Would you like me to add it?

https://github.com/Opentrons/opentrons/blob/28d54abb20a5d8b706ec64d5c32e044317e71c57/shared-data/command/types/pipetting.ts#L203-L207

Comment (SyntaxColoring) [reply app/src/molecules/Command/hooks/useCommandTextString/index.tsx:81]: Yes please, sorry about that.

**Review by ecormany — APPROVED** [review]
Command text looks good. Is this command text supposed to be emitted when doing an aspirate with `liquid_presence_detection=True` set for the pipette? Because I'm not seeing that. (I also can't get standalone `pipette.detect_liquid_presence()` to work but that seems to be a separate bug.)

File: app/src/molecules/Command/hooks/useCommandTextString/utils/getLiquidProbeCommandText.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  LiquidProbeRunTimeCommand,
+  RunTimeCommand,
+} from '@opentrons/shared-data'
+import type { HandlesCommands } from './types'
+import {
+  getFinalLabwareLocation,
+  getLabwareDisplayLocation,
+  getLabwareName,
+} from '../../../utils'
+import type { TFunction } from 'i18next'
```
Comment (jerader) [inline app/src/molecules/Command/hooks/useCommandTextString/utils/getLiquidProbeCommandText.ts:13]: nit import order can go higher

**Review by jerader — APPROVED** [review]
makes sense to me! `liquidProbe` and `tryLiquidProbe` are supposed to be part of schema 9 right?

**Comment by mjhuff** [comment]
> Command text looks good. Is this command text supposed to be emitted when doing an aspirate with `liquid_presence_detection=True` set for the pipette? Because I'm not seeing that. (I also can't get standalone `pipette.detect_liquid_presence()` to work but that seems to be a separate bug.)

My understanding is this is an entirely separate command that is emitted before an aspirate. CC @SyntaxColoring 

> makes sense to me! `liquidProbe` and `tryLiquidProbe` are supposed to be part of schema 9 right?

I'm like 90% sure this is correct but I'll  CC @SyntaxColoring just to be safe.

**Comment by SyntaxColoring** [comment]
> Is this command text supposed to be emitted when doing an aspirate with `liquid_presence_detection=True` set for the pipette?

Yes. The `aspirate()` in the user's Python script should emit a `liquidProbe` Protocol Engine command, which will get its own command text, and then emit an `aspirate` Protocol Engine command, which will also get its own command text.

> > makes sense to me! `liquidProbe` and `tryLiquidProbe` are supposed to be part of schema 9 right?
> 
> I'm like 90% sure this is correct but I'll CC @SyntaxColoring just to be safe.

Yep.

**Pull request merged**